Assets/Scripts/GameManager.cs

```diff
@@ -19,16 +20,36 @@ public class GameManager : MonoBehaviour
      //private bool transitionStarted = false;
 
      private int frameCount = 0;
-
+    
 
     Text statusText;
 
+    public static int level;
+    public GameObject ballPrefab;
```

Comment:
Could we dynamically load this resource, instead of injecting it statically?
e.g. `Resources.load(...)`